tests/installation/releasenotes.pm

```diff
@@ -56,11 +56,16 @@ sub run() {
 
     # no relnotes for ltss in QAM_MINIMAL
     push @no_relnotes, qw(ltss) if get_var('QAM_MINIMAL');
+    # no HA-GEO release-notes for SLE12-SP1 on s390x
+    if (check_var('ARCH', 's390x') and check_var('VERSION', '12-SP1')) {
+        push @no_relnotes, qw(geo);
+        record_soft_failure 'bsc#1033504';
```

Comment:
https://bugzilla.suse.com/show_bug.cgi?id=1033504#c4 states correctly that for s390x we would need the release notes on s390x media which is not going to be updated so the bug will never be fixed. Just accept that there are no release notes. You can still reference the bug in the comment though